src/vs/editor/contrib/suggest/suggestWidget.ts

```diff
@@ -458,6 +458,7 @@ export class SuggestWidget implements IContentWidget, IVirtualDelegate<ICompleti
 			editor.onDidLayoutChange(() => this.onEditorLayoutChange()),
 			this.list.onSelectionChange(e => this.onListSelection(e)),
 			this.list.onFocusChange(e => this.onListFocus(e)),
+			this.list.onMouseOver(e => this.list.setFocus([e.index])),
```

Comment:
@ramya-rao-a this adds focus on mouseover to all suggest widgets. This makes sense to me but what do you think? I could put it behind a flag somewhere.